extensions/ui/org.eclipse.smarthome.ui.paper/web-src/js/controllers/controller.things.view.js

```diff
@@ -78,15 +78,14 @@ angular.module('PaperUI.controllers.things') //
 
     $scope.linkChannel = function(channelID, event, preSelect) {
         var channel = $scope.getChannelById(channelID);
-        var channelType = $scope.getChannelTypeByUID(channel.channelTypeUID);
 
         var params = {
             linkedItems : channel.linkedItems.length > 0 ? channel.linkedItems : '',
             acceptedItemTypes : channel.acceptedItemTypes,
-            category : channelType.category ? channelType.category : '',
-            suggestedName : getItemNameSuggestion(channelID, channelType.label),
-            suggestedLabel : channelType.label,
-            suggestedCategory : channelType.category ? channelType.category : '',
+            category : channel.category ? channel.category : '',
+            suggestedName : getItemNameSuggestion(channelID, channel.label),
+            suggestedLabel : channel.label,
+            suggestedCategory : channel.category ? channel.category : '',
```

Comment:
see above

Reply:
Thats right, the category of an channel is governed by the channelType.
I can fix the fix - or makes it sense to specify the category at the channel too? - I'm not that deep into it.